calibration/DOFSTK/CGINS-DOFSTK-00209__20200718.csv

```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-0209,CC_frequency_offset,-7.876420e+002,source_file DOFST-K_SBE43F_SN_0209_Calibration_2020-07-18.zip > 0209.cal and SBE 43F O0209 SOCAdjusted 23Jul20.pdf
+43-0209,CC_oxygen_signal_slope,2.9150e-004,use adjusted Soc
```

Comment:
All values look good.  Need to be sure this slope ('adjusted Soc') value is in the instrument.